superset-frontend/src/components/FlashError.jsx

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
Could you make this a `tsx` file since we're trying to have all new files use TypeScript?